packages/eslint-plugin-gestalt/src/prefer-box-no-disallowed.test.js

```diff
@@ -0,0 +1,149 @@
+// @flow strict
```

Comment:
this file isn't new, got deleted in a previous PR by mistake. Bringing it back to life.

Reply:
![giphy-3](https://user-images.githubusercontent.com/12059539/153685891-1905e5f2-d2cc-4ffb-a1f9-46d5d1ced611.gif)